core/models/project.py

```diff
@@ -219,3 +222,62 @@ class FundType(models.TextChoices):
 
     def __str__(self):
         return f"{self.project.title} {self.amount} {self.date}"
+
+
+class ProjectProgressReport(models.Model):
```

Comment:
Make sure that the admin pages of the linked models (Agency, Country, etc.) are working (add projectprogressreport field in the exclude list from get_list_display of each admin model)

Reply:
Aha, did not noticed that all ModelAdmins automatically use all available fields for listing. Will add the exceptions where needed.